terraform/environments/cooker/test-ec2/main.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
```

Comment:
We already have providers for cooker here https://github.com/ministryofjustice/modernisation-platform/blob/main/terraform/environments/cooker/providers.tf